client/lib/application/src/application-list/application-list.hbs

```diff
@@ -3,6 +3,19 @@
     <a href="/application/create" class="btn btn-default">
     <i class="fa fa-plus"></i> Create Application</a>
 </div>
+<form>
+    <input
+        autofocus="autofocus"
+        value="{{filterTerm}}"
+        type="search"
+        aria-label="Enter your filter term"
+        placeholder="Application" />
+    <button
+        type="submit"
+        class="btn btn-default">
+            <i class="fa fa-filter"></i> Filter
```

Comment:
Since we’re not using the term `filter` anywhere else in the application I propose to call it `search`.